backend/lib/endpoints/feedback.js

```diff
@@ -0,0 +1,36 @@
+const { createFeedbackSchema } = require("./schema/feedback");
+
+/*
+ * /api/feedback
+ */
+async function routes(app) {
+  const Feedback = app.mongo.model("Feedback");
+
+  app.post("/", { schema: createFeedbackSchema }, async (req, reply) => {
+    const { userId } = req.body;
+
+    if (userId) {
+      const [err, userFeedback] = await app.to(Feedback.find({ userId }));
+      if (userFeedback[0] || err) {
+        throw app.httpErrors.internalServerError();
```

Comment:
ok this is the same as my other recent changes...
I think this might be the best one, even if it's not as shorts as using `reply.notFound()` etc. but because we also `throw` then it's clear that it's actually an error ... well just a nitpick,

Reply:
It does make more sense that it is immediately recognizable as an error due to `throw` being there. `reply` could theoretically be anything.